node/src/effect/requests.rs

```diff
@@ -382,6 +382,24 @@ pub enum ApiRequest<I> {
         responder: Responder<Result<QueryResult, engine_state::Error>>,
     },
     /// Query the global state at the given root hash.
+    QueryEraValidators {
+        /// The global state hash.
```

Comment:
Comment out of date with field name.

Reply:
The comment is correct and explanatory; `state_root_hash` is a global state hash.